packages/marshal/test/test-marshal-capdata.js

```diff
@@ -188,6 +189,45 @@ test('unserialize extended errors', t => {
   console.log('error with extra prop', unkErr);
 });
 
+test('unserialize errors w recognized extensions', t => {
+  if (typeof AggregateError === 'undefined') {
+    t.pass('skip test on platforms prior to AggregateError');
+    return;
+  }
+  const { unserialize } = makeTestMarshal();
+  const uns = body => unserialize({ body, slots: [] });
+
+  const errEnc = '{"@qclass":"error","message":"msg","name":"URIError"}';
+
+  const refErr = uns(
+    `{"@qclass":"error","message":"msg","name":"ReferenceError","extraProp":"foo","cause":${errEnc},"errors":[${errEnc}]}`,
+  );
+  t.is(getPrototypeOf(refErr), ReferenceError.prototype); // direct instance of
+  t.false('extraProp' in refErr);
+  t.is(getPrototypeOf(refErr.cause), URIError.prototype);
+  t.is(getPrototypeOf(refErr.errors[0]), URIError.prototype);
+  console.log('error with extra prop', refErr);
+
+  const aggErr = uns(
+    `{"@qclass":"error","message":"msg","name":"AggregateError","extraProp":"foo","cause":${errEnc},"errors":[${errEnc}]}`,
+  );
+  t.is(getPrototypeOf(aggErr), AggregateError.prototype); // direct instance of
+  t.false('extraProp' in aggErr);
+  t.is(getPrototypeOf(aggErr.cause), URIError.prototype);
+  t.is(getPrototypeOf(aggErr.errors[0]), URIError.prototype);
+  console.log('error with extra prop', aggErr);
+
+  const unkErr = uns(
+    `{"@qclass":"error","message":"msg","name":"UnknownError","extraProp":"foo","cause":${errEnc},"errors":[${errEnc}]}`,
+  );
+  t.is(getPrototypeOf(unkErr), Error.prototype); // direct instance of
+  t.false('extraProp' in unkErr);
+  t.is(getPrototypeOf(unkErr.cause), URIError.prototype);
+  t.is(getPrototypeOf(unkErr.errors[0]), URIError.prototype);
+
+  console.log('error with extra prop', unkErr);
```

Comment:
t.log doesn't do anything relevant. The point was to see the annotations in the SES `console` output, which isn't appropriate in an automated test anyway. These SES console issues were adequately covered by the SES tests using `throwsAndLogs` for automated `console` testing, and the `test-*-console-demo.js` files to visually inspect the SES `console` output. So, here, I just deleted all these `console.log` lines rather than revising them.

Done.